### ISSUE TITLE
mrp_parallel potential new module - Need some help

### ISSUE THREAD

**Issue by chrisandrewmann** [OCA/manufacture#394]
Hi all,

I have written a module that i’m hoping to submit for manufacturing.
I’m very new to this and am still learning Odoo and Python, along with understanding the OCA community.
Currently the module is in my newly setup Github account here: https://github.com/chrisandrewmann/mrp_parallel
I wrote this to satisfy the needs of the company I work for, in about 3 weekends during my spare time and was hoping it would be useful for others to expand upon.
All details are on my Github with some screenshots and a readme.
 
I've tried to follow the steps as closely as I can to make it compatible with OCA, but i'm looking for help to get it ready to use as I have limited time.
Would you mind taking a look and first reviewing if it would be useful and accepted?

Thanks
Chris

**Comment by LoisRForgeFlow** [comment]
Hi @chrisandrewmann 

It seems you put together some good features there! and I think your work will be welcomed to OCA repositories. My advice here is:

1. Split it into different features: "Engineering Spec Sheet" can be one module, "Parallel manufacturing work centers" can be another and "Timeline view for Work" can be a third one for example. The benefit is that you can maximize adoption and usability, due to there must be people interested for example in the parallel work centers but not in the timeline view (as they might have their own view or use the entreprise's). As a general advice, modules at OCA should be as abstract as possible, so try to see if there are things very specific to your use case and think in the bigger picture.

2. Go over the guidelines and try to apply it to your modules: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst. The template module could also be helpful for you: https://github.com/OCA/maintainer-tools/tree/master/template/module (at least you can copy from there the readme folder and fill information there, it will be used to generate the README file once a module is merged in OCA).

Please let me know if you need any clarification in the process.

**Comment by chrisandrewmann** [comment]
Hi there, 
Thanks for your reply. Will take note of that and when time allows, try to split into 3 parts.
Will also look at the template and aim to make it fit better. No sure when as very busy currently.
Regards,
Chris
